lib/fpm/command.rb

```diff
@@ -233,6 +233,17 @@ def help(*args)
     "copying, downloading, etc. Roughly any scratch space fpm needs to build " \
     "your package.", :default => Dir.tmpdir
 
+  option "--source-date-epoch", "SOURCE_DATE_EPOCH",
+    "Integer unix timestamp to use for generated files. " \
+    "See https://reproducible-builds.org/specs/source-date-epoch",
+    :environment_variable => "SOURCE_DATE_EPOCH"
+
+  option "--build-path-prefix-map", "BUILD_PATH_PREFIX_MAP",
+    "If SOURCE_DATE_EPOCH is given without BUILD_PATH_PREFIX_MAP, " \
```

Comment:
That's because I wrote it before I figured out what it'd do :-)   I'll add that soon.  Bit hard to test without a compiler that supports it, so maybe I'll have to build one.